test/core/tsi/s2a/s2a_test_util.cc

```diff
@@ -17,10 +17,84 @@
  */
 
 #include "test/core/tsi/s2a/s2a_test_util.h"
+#include <grpc/support/log.h>
+#include <grpc/support/string_util.h>
 #include "src/core/tsi/s2a/record_protocol/s2a_crypter.h"
 #include "src/core/tsi/s2a/record_protocol/s2a_crypter_util.h"
 #include "src/core/tsi/s2a/s2a_constants.h"
 
+uint8_t aes_128_gcm_key_bytes[17] = {0xc3, 0xae, 0x75, 0x09, 0xcf, 0xce,
```

Comment:
Can you please add a comment about how these vectors were generated in case we need to regenerate them in the future.

Reply:
Done.